pkg/landscaper/installations/scope.go

```diff
@@ -0,0 +1,130 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package installations
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	lsv1alpha1helper "github.com/gardener/landscaper/apis/core/v1alpha1/helper"
+	"github.com/gardener/landscaper/pkg/landscaper/operation"
+)
+
+// Scope contains the visible installations of a specific installation.
+// This scope is later used to validate and get import data
+type Scope struct {
+	// Name is the name of the current installation's context.
+	// BY default it is the source name of the parent.
+	Name string
+	// Parent is the installation the installation is encompassed in.
+	// Parents are handled separately as installation have access to the same imports as their parent.
+	Parent *Installation
+
+	// Siblings are installations with the same parent.
+	// The installation has access to the exports of theses components
+	Siblings []*InstallationBase
+}
+
+// SetInstallationScope determines the current context and updates the operation context.
+func (o *Operation) SetInstallationScope(ctx context.Context) error {
+	newCtx, err := o.DetermineInstallationScope(ctx)
+	if err != nil {
+		return err
+	}
+	o.scope = *newCtx
+	return nil
+}
```

Comment:
The usage of context and scope is a little bit confusing here. What is the difference?